### DISCUSSION TITLE
Installing/removing packages puts Prisma types out of sync

### DISCUSSION THREAD

**Discussion by axhamre** [prisma/prisma#12979] [Q&A]
Every time I install or remove a npm package, I need to run `prisma generate` afterwards (see video).

1. Is this intended behaviour?
2. If yes, does anyone know of any workarounds?

https://user-images.githubusercontent.com/1907343/164981485-f40f9d40-9bba-4519-9f37-6ba55252db48.mp4

**Comment by austincrim** [comment]
That's definitely unexpected! 
- What version of Yarn are you using?
- What's your repo setup look like? 
- Are you using a monorepo? Yarn workspaces?

**Comment by axhamre** [comment]
So I'm using Yarn `1.22.15`. And the repo is not a monorepo, the setup is farily straight forward - very similar to this: https://github.com/hayes/pothos/tree/main/examples/prisma

**Reply by austincrim** [reply]
I was unable to reproduce this with the same Yarn version and repo setup. If you can create a minimal reproduction repo, please file a bug report with the repo linked and our engineers will take it from there 👍.

One random thing you might try in the meantime, when you get the type error, try running the `TypeScript: Restart TS Server` command in VS Code with <kbd>CMD/CTRL + Shift + P</kbd>.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
